docs/source/guides/speed.rst

```diff
@@ -0,0 +1,301 @@
+.. testsetup:: *
+
+    from pytorch_lightning.trainer.trainer import Trainer
+    from pytorch_lightning.callbacks.early_stopping import EarlyStopping
+    from pytorch_lightning.core.lightning import LightningModule
+
+.. _speed:
+
+#######################
+Speed up model training
+#######################
+
+There are multiple ways you can speed up your model's time to convergence:
+
+* `<Early stopping_>`_
+
+* `<GPU/TPU training_>`_
+
+* `<Mixed precision (16-bit) training_>`_
+
+* `<Control Training Epochs_>`_
+* `<Control Validation Frequency_>`_
+
+* `<Limit Dataset Size_>`_
+
+.. _early_stopping:
+
+**************
+Early stopping
+**************
+
+**Use when:**
+
+.. raw:: html
+
+    <video width="50%" max-width="400px" controls
+    poster="https://pl-bolts-doc-images.s3.us-east-2.amazonaws.com/pl_docs/trainer_flags/yt_thumbs/thumb_earlystop.png"
+    src="https://pl-bolts-doc-images.s3.us-east-2.amazonaws.com/pl_docs/yt/Trainer+flags+19-+early+stopping_1.mp4"></video>
+
+|
+
+Early stopping is an optimization technique used to avoid overfitting without compromising the model's accuracy. When training a large network, there will be a point during training when the model will stop generalizing and start learning the statistical noise in the training dataset. To avoid overfitting and reduce training time, unable early stopping to stop training at the point when performance on a validation dataset starts to degrade (you can pick the metric to monitor).
```

Comment:
```suggestion
Early stopping is an optimization technique used to avoid overfitting without compromising the model's accuracy. When training a large network, there will be a point during training when the model will stop generalizing and start learning the statistical noise in the training dataset. To avoid overfitting and reduce training time, enable early stopping to stop training at the point when performance on a validation dataset starts to degrade (you can pick the metric to monitor).
```